packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -91,7 +91,8 @@ const getPublishedAtClause = (
    * As it only contains entries with publishedAt set.
    */
   if (!contentTypes.hasDraftAndPublish(targetModel)) {
-    return { $ne: null };
+    return {};
```

Comment:
to be discussed

Reply:
makes sense! 

just wondering, does it make any difference ?

Reply:
yeah, internal code u&p for ex is creating the default roles without a publishedAt as the field doesn't have a default. 
On v5 release this will break for people with existing data. I don't think it's worth trying to migrate everything to have one when D&P is disabled.

Reply:
isn't the publishedAt value set to a date by defualt on the db layer?